02-peer-protocol.md

```diff
@@ -277,24 +272,24 @@ The receiving node:
     - the `serial_id` has the wrong parity
     - it has received 4096 `tx_add_output` messages during this negotiation
     - the `sats` amount is less than the `dust_limit`
-    - the `sats` amount is greater than 210,0000,000,000,000(`MAX_MONEY`)
-
+    - the `sats` amount is greater than 2,100,000,000,000,000 (`MAX_MONEY`)
 
 #### Rationale
+
 Each node must know the set of the transaction outputs.
 
 `serial_id` is a randomly chosen number which uniquely identifies this output.
 Outputs in the constructed transaction are sorted by `serial_id`.
 
 `sats` is the satoshi value of the output.
 
-`script` is the scriptPubKey for the output. The length is omitted.
```

Comment:
Wait, why are we removing this comment? Haha.

Reply:
Because the length isn't omitted, is it? Since we have the `scriptlen` field? Or did I misunderstand what `length` refers to here?

Reply:
Ah, yeah if you're pulling the `scriptPubKey` field out of the transaction, you wouldn't include the length data/byte.

This is a departure from the `scriptPubKey` field inclusion in the sighash algos, if I recall correctly, so it seemed worth pointing out.

E.g. we'd expect `0020.....` for a P2WSH not `220020...`

Reply:
Gotcha, I now understand why you put that up. We already encode `scriptPubKey` fields when doing `upfront_shutdown_script`, and we didn't specify that the length is omitted, so I guess we don't need to specify it here? People implemented this correctly for `upfront_shutdown_script` :)